device.h

```diff
@@ -11,6 +11,15 @@
 #define PIXFMTS_MAX 4
 #define FB_NAME_MAXLENGTH 16
 
+#define LD_360_WIDTH 480
```

Comment:
It is not worthy to define the values here. Use V4L2 official macros instead.

Reply:
In v4l2 offical macros, it doesn't define 360, but it define 480 and 720.

Reply:
Then, use V4L2 macros along with the additional definitions.